.github/workflows/e2e-pw-pull-request.yml

```diff
@@ -0,0 +1,77 @@
+name: E2E Playwright Tests - Pull Request
+
+on:
+    pull_request:
+        branches:
+            - develop
+            - trunk
+    workflow_dispatch:
+    workflow_call:
+        inputs:
+            wcpay-use-build-artifact:
+                type: boolean
+                required: false
+                default: false
+            repo-branch:
+                type: string
+                required: false
+                description: 'Branch to be used for running tests'
+
+env:
+    E2E_GH_TOKEN: ${{ secrets.E2E_GH_TOKEN }}
+    WCP_DEV_TOOLS_REPO: ${{ secrets.WCP_DEV_TOOLS_REPO }}
+    WCP_DEV_TOOLS_BRANCH: 'trunk'
+    WCP_SERVER_REPO: ${{ secrets.WCP_SERVER_REPO }}
+    WC_SUBSCRIPTIONS_REPO: ${{ secrets.WC_SUBSCRIPTIONS_REPO }}
+    E2E_BLOG_ID: ${{ secrets.E2E_BLOG_ID }}
+    E2E_BLOG_TOKEN: ${{ secrets.E2E_BLOG_TOKEN }}
+    E2E_USER_TOKEN: ${{ secrets.E2E_USER_TOKEN }}
+    WC_E2E_SCREENSHOTS: 1
+    E2E_SLACK_CHANNEL: ${{ secrets.E2E_SLACK_CHANNEL }}
+    E2E_SLACK_TOKEN: ${{ secrets.E2E_SLACK_TOKEN }}
+    E2E_USE_LOCAL_SERVER: false
+    E2E_RESULT_FILEPATH: 'tests/e2e/results.json'
+    WCPAY_USE_BUILD_ARTIFACT: ${{ inputs.wcpay-use-build-artifact }}
+    WCPAY_ARTIFACT_DIRECTORY: 'zipfile'
+    NODE_ENV: 'test'
+    FORCE_E2E_DEPS_SETUP: true
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+
+jobs:
+    wcpay-e2e-tests:
+        runs-on: ubuntu-latest
+
+        name: WC - latest
+
+        env:
+            E2E_WP_VERSION: 'latest'
+            E2E_WC_VERSION: 'latest'
+
+        steps:
+            - name: Checkout WCPay repository
+              uses: actions/checkout@v4
+              with:
+                  ref: ${{ inputs.repo-branch || github.ref }}
+
+            - name: 'Download WooCommerce Payments build file'
+              if: ${{ inputs.wcpay-use-build-artifact }}
+              uses: actions/download-artifact@v4
+              with:
+                  name: woocommerce-payments
+                  path: ${{ env.WCPAY_ARTIFACT_DIRECTORY }}
+
+            - name: Setup E2E environment
+              uses: ./.github/actions/e2e/env-setup
+
+            - name: Install Playwright
+              shell: bash
+              run: |
+                  echo "::group::Install Playwright"
+                  npx playwright install
+                  echo "::endgroup::"
```

Comment:
This step currently installs more browsers than just Chromium – might be wasteful unless we intend to test against the other browsers.

<img width="1091" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/57c98222-7a24-483c-baf1-2b41ccb6f045">

Reply:
I also noticed it on local. We can be more specific and run it like `npx playwright install chromium` to ensure it only installs a single browser.

Reply:
Resolved in fd60c3780d461d408b1f7789ed3f16e0002960fe

<img width="1052" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/8430718e-f811-4cda-b832-7a9fda16583c">